src/Libraries/OrchardCore.Commerce.Abstractions/Abstractions/ICheckoutViewModel.cs

```diff
@@ -14,9 +14,10 @@ public interface ICheckoutViewModel : IPaymentViewModel, IShape
 {
     string ShoppingCartId { get; }
     Amount GrossTotal { get; }
-    IEnumerable<SelectListItem> Regions { get; set; }
+    IEnumerable<Region> Regions { get; set; }
```

Comment:
Why is this necessary? Such breaking changes should be avoided, because it will cause errors for anyone who customized their `Checkout` shape for any reason. I know of at least two such cases in client projects, so this not a speculative concern.

I suggest creating a separate new property if you need to reference the original region data as well. For example:

```csharp
    IEnumerable<SelectListItem> Regions { get; set; }
    IEnumerable<Region> RegionData { get; set; }
```